source/includes/contrib.md

```diff
@@ -1,15 +1,10 @@
 # Contributing
 
-Help us keep this documentation up to date by forking 
+Please help us keep this documentation up to date by forking 
 [this repository](https://github.com/openmrs/openmrs-contrib-rest-api-docs) 
-and submitting pull requests with improvements or corrections. The quality 
-of our documentation depends on the collective contributions of persons just 
-like you. Thank you for your contributions!
+and submitting pull requests with improvements or corrections. The quality of our documentation depends on the collective contributions of persons just like you. Thank you for your contributions!
```

Comment:
am not sure why do you made these changes. was there any indentation problem because of these line lengths ?

Reply:
actually I thought these are unnecessary endlines so I removed them, I will revert back If you want